app/javascript/components/organisms/TabsSection.js

```diff
@@ -2,13 +2,13 @@ import React from 'react'
 import TabsMenu from '../molecules/TabsMenu'
 import TabCard from './TabCard'
 
-const TabsSection = ({ children, background, padding, ...props }) => {
+const TabsSection = ({ children, background, padding, tabBackground, ...props }) => {
```

Comment:
is `tabBackground` ever being passed in? Seems like we're just defaulting to white now?

If the tab background is always going to be white I'd rather we just remove the prop options because the way it's done now is not my favorite (the prop is a string that needs to turn into a valid class when `bg-` is prepended, just seems fragile).

Reply:
yeah I think that's reasonable. I thought we'd be using it in different contexts but we're not